### DISCUSSION TITLE
I have a phone with a dead battery, can I use it?

### DISCUSSION THREAD

**Discussion by Forowebmasters** [DeviceFarmer/stf#705] [Q&A]
It turns on with 1% battery all the time,
I don't know if at some point it will stop working at 100%.
It has not exploded, it has only deteriorated, this could be used with this process?

I want it to have it as a 24/7 WhatsApp customer service center.

**Comment by denis99999** [comment]
@DavidMoralesMx , if your phone is corrupted I recommend you change it to avoid aggravation like at worst the explosion of the phone.
